frontend/src/views/Collection/index.tsx

```diff
@@ -65,7 +73,14 @@ const Collection: FC = () => {
     visibility,
   });
 
-  const { data: collection, isError, isFetching } = collectionState;
+  const { data, isError, isFetching } = collectionState;
+
+  const [deleteMutation] = useDeleteCollection(id, visibility);
+
+  if (data?.tombstone === true) {
+    router.push(ROUTES.HOMEPAGE + "?tombstoned_dataset_id=" + id);
+  }
+  const collection = data as CollectionType;
```

Comment:
just curious -- is there a reason to do this than keeping `const { data: collection, isError, isFetching } = collectionState;` ?

Reply:
I thought it better expressed that once we get past the tombstone check we actually know its a collection

Reply:
ahh I see! I was thinking tombstoned collection is still a collection, and we're already extracting `data` from `collectionState`, so the association is already there, so we could probably save an assignment in a new line!